### PR TITLE
release-tools: add support for NEWS.md changelog.py

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: run-checks [hunk 1]
```diff
@@ -378,6 +378,7 @@ if [ "$UNIT" = 1 ]; then
     command -v python3 && python3 ./tests/lib/tools/mountinfo.query --run-unit-tests
     command -v python2 && python2 ./tests/lib/tools/version-compare --run-unit-tests
     command -v python3 && python3 ./tests/lib/tools/version-compare --run-unit-tests
+    command -v pytest-3 && pytest-3 ./release-tools
```
Comment (ZeyadYasser) [inline run-checks:381]: Some packages needs to be installed for tests to work properly that are not installed by default `bs4`, `flake8`, on a fresh virtual environment I had to install `bs4`, `markdown`, `python-debian` and `flake8`.

Do you think we could/should skip the test gracefully if those modules are not installed?

**Review by ZeyadYasser — APPROVED** [review]
This is a very nice addition, Thank you!

File: run-checks (hunk 1 above)
Comment (ZeyadYasser) [inline run-checks:381]: I re-ran failing unit tests and it looks like we actually need to install those dependencies in the github workflow runner.

Comment (ZeyadYasser) [reply run-checks:381]: something like maybe
```diff
diff --git a/.github/workflows/test.yaml b/.github/workflows/test.yaml
index 68ffbf099f..2221eef349 100644
--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -246,6 +246,11 @@ jobs:
           sudo apt update
           sudo apt build-dep -y ${{ github.workspace }}/src/github.com/snapcore/snapd
 
+    - name: Install Pytest dependencies
+      run: |
+          sudo apt update
+          sudo apt install python3-pytest flake8
+
     # golang latest ensures things work on the edge
     - name: Install the go snap
```

**Review by ZeyadYasser — APPROVED** [review]
LGTM, Thank you!

**Review by ernestl — APPROVED** [review]
We have successfully used this for 2.61 and 2.61.1.
I created a followup PR for to propose small changes to improve user experience: https://github.com/snapcore/snapd/pull/13369

**Pull request merged**